api/README.md

```diff
@@ -123,41 +183,54 @@ pushd ../engines/router
 popd
 ```
 
+#### <b>Start Turing API</b>
+---
 Now fire up the turing API server (as a daemon or tmux or some other terminal process).
 
 ```
 go run turing/cmd/main.go -config=config-dev.yaml
 ```
 
-Now run E2E tests.
-
+#### <b>Run E2E tests</b>
+---
 ```
-TEST_ID=$(date +%Y%m%d%H%M) \
```

Comment:
Is it ok to remove `TEST_ID`? I see it's still used here and it also has some README notes: `api/e2e/new-test/config/config.go`

Reply:
It's ok because by default, `DefaultConfig()` in `api/e2e/new-test/config/config.go` will be triggered first when loading the configs.